fix(faq, settings, notification): resolve overlapping screens during transitions

biplab1 opened the pull request:
Jira Task: [MW-297](https://mifosforge.jira.com/browse/MW-297)

## Screenshots

Before Fix:

[Before-Fix-overlapping-screens-transition.webm](https://github.com/user-attachments/assets/12f9789d-dfe2-47f7-8db7-7a88b5b8c5aa)

After Fix:

[After-Fix-overlapping-screens-transition.webm](https://github.com/user-attachments/assets/6af3cfdb-c4a4-4110-8c10-a8d48d6b9377)

## Description
 - Fixed overlapping screens during transition

##
<!--Please make sure these boxes are checked before submitting your pull request - thanks!-->

- [ ] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ ] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them.


[MW-297]: https://mifosforge.jira.com/browse/MW-297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

biplab1 committed:
fix(faq, settings): resolve overlapping screens during transitions

HekmatullahAmin commented on feature/faq/src/commonMain/kotlin/org/mifospay/feature/faq/FaqScreen.kt:
```diff
@@ -80,39 +80,43 @@ private fun FaqScreen(
-                .weight(1f)
-                .fillMaxWidth(),
+                .fillMaxSize()
+                .padding(contentPadding),
```
- the `Column` around the `LazyColumn` isn’t needed here. u can pass the scaffold’s contentPadding straight into the list and let it fill the available space.
- and add `rememberLazyListState()` to preserve scroll.

```
val listState = rememberLazyListState()

        LazyColumn(
            state = listState,
            modifier = Modifier.fillMaxSize(),
            contentPadding = contentPadding,
        ) {
```

biplab1 replied:
Got it, I’ll remove the extra `Column` and use `rememberLazyListState` to preserve scroll.

biplab1 committed:
refactor(feature:faq): remove redundant Column and add listState

biplab1 commented:
@niyajali I’ve addressed the review comment. Please take a look when you get a chance. Let me know if anything else is needed, otherwise kindly proceed with the merge.

biplab1 commented:
@niyajali Also, @therajanmaurya asked me to submit fixes for these screens only. Someone else will pick up the ticket to fix the issue on the remaining screens.

niyajali committed:
Merge branch 'development' into MW-297-overlapping-screens-during-transition
